packages/cice5/package.py

```diff
@@ -18,8 +13,14 @@ class Cice5(MakefilePackage):
     maintainers("harshula", "anton-seaice")
     license("BSD-3-Clause", checked_by="anton-seaice")
 
-    version("access-om2", branch="master", preferred=True)
-    version("access-esm1.6", branch="access-esm1.6")
```

Comment:
Hi @anton-seaice , Should we keep version `access-esm1.6` for now?

Reply:
@anton-seaice & I had a chat. These two lines are not going to be deleted in this PR.